src/test/resources/test_data/overall_calculation/custom-examples/crs-1070-ual-on-tused.json

```diff
@@ -0,0 +1,355 @@
+        "isPcscSds":false,
+        "committedAt":"2021-05-31",
+        "offenceCode":"PC53001",
+        "isPcscSec250":false,
+        "isPcscSdsPlus":false,
+        "isScheduleFifteen":false,
+        "isScheduleFifteenMaximumLife":false
+      },
+      "duration":{
+        "durationElements":{
+          "DAYS":145,
+          "WEEKS":0,
+          "YEARS":0,
+          "MONTHS":0
+        }
+      },
+      "identifier":"75833df9-a090-3648-b54f-9ab3b6133f95",
+      "recallType":"STANDARD_RECALL",
+      "section250":false,
+      "sentencedAt":"2022-02-24",
+      "caseSequence":1,
+      "lineSequence":2,
+      "caseReference":"STD LR",
+      "consecutiveSentenceUUIDs":[
+        "f36bc182-5c02-3cc3-a4bf-df55a1a41fb4"
+      ]
+    },
+    {
+      "type":"AFineSentence",
+      "offence":{
+        "isPcscSds":false,
+        "committedAt":"2022-04-25",
+        "offenceCode":"TH68010",
+        "isPcscSec250":false,
+        "isPcscSdsPlus":false,
+        "isScheduleFifteen":false,
+        "isScheduleFifteenMaximumLife":false
+      },
+      "duration":{
+        "durationElements":{
+          "DAYS":25,
+          "WEEKS":0,
+          "YEARS":0,
+          "MONTHS":0
+        }
+      },
+      "fineAmount":919.72,
+      "identifier":"f9c7ff8a-25ba-39ba-9967-77f94bcdfc16",
+      "recallType":null,
+      "sentencedAt":"2022-04-25",
+      "caseSequence":4,
+      "lineSequence":3,
+      "caseReference":"16008673W CL",
+      "consecutiveSentenceUUIDs":[
+      ]
+    },
+    {
+      "type":"AFineSentence",
+      "offence":{
+        "isPcscSds":false,
+        "committedAt":"2022-04-25",
+        "offenceCode":"TH68010A",
+        "isPcscSec250":false,
+        "isPcscSdsPlus":false,
+        "isScheduleFifteen":false,
+        "isScheduleFifteenMaximumLife":false
+      },
+      "duration":{
+        "durationElements":{
+          "DAYS":25,
+          "WEEKS":0,
+          "YEARS":0,
+          "MONTHS":0
+        }
+      },
+      "fineAmount":919.72,
+      "identifier":"f9c7ff8a-25ba-39ba-9967-77f94bcdfc16",
+      "recallType":null,
+      "sentencedAt":"2022-04-25",
+      "caseSequence":4,
+      "lineSequence":3,
+      "caseReference":"16008673W CL",
+      "consecutiveSentenceUUIDs":[
+      ]
+    },
+    {
+      "type":"AFineSentence",
+      "offence":{
+        "isPcscSds":false,
+        "committedAt":"2022-04-25",
+        "offenceCode":"TH68026",
+        "isPcscSec250":false,
+        "isPcscSdsPlus":false,
+        "isScheduleFifteen":false,
+        "isScheduleFifteenMaximumLife":false
+      },
+      "duration":{
+        "durationElements":{
+          "DAYS":25,
+          "WEEKS":0,
+          "YEARS":0,
+          "MONTHS":0
+        }
+      },
+      "fineAmount":919.72,
+      "identifier":"f9c7ff8a-25ba-39ba-9967-77f94bcdfc16",
+      "recallType":null,
+      "sentencedAt":"2022-04-25",
+      "caseSequence":4,
+      "lineSequence":3,
+      "caseReference":"16008673W CL",
```

Comment:
You may want to remove all the data here that isn't required and also isn't anonymous.

Reply:
Thanks